samples/AspireWithMasstransit/AspireWithMasstransit.ApiService/Program.cs

```diff
@@ -0,0 +1,34 @@
+using System.Reflection;
+using AspireWithMasstransit.ApiService;
+using MassTransit;
+
+var builder = WebApplication.CreateBuilder(args);
+
+// Add service defaults & Aspire components.
+builder.AddServiceDefaults();
+
+var connectionString = builder.Configuration.GetConnectionString("messaging");
+        
+builder.Services.AddMassTransit(s =>
+{
+    s.AddConsumers(Assembly.GetExecutingAssembly());
+    s.UsingRabbitMq((context, cfg) =>
+    {
+        cfg.Host(new Uri(connectionString!));
```

Comment:
Rather than disabling the NRT warning here with `!`, make line 10 use a `throw` expression in case the connection string is `null`, e.g.

```suggestion
var connectionString = builder.Configuration.GetConnectionString("messaging") ?? throw new InvalidOperationException("Connection string for RabbitMQ instance 'messaging' was not found.");
        
builder.Services.AddMassTransit(s =>
{
    s.AddConsumers(Assembly.GetExecutingAssembly());
    s.UsingRabbitMq((context, cfg) =>
    {
        cfg.Host(new Uri(connectionString));
```